src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.Edit.cshtml

```diff
@@ -109,8 +109,8 @@
 
                 @if (IdentityOptions.Value.Password.RequiredUniqueChars > 1)
                 {
-                    // The generated password does not meet the required-unique-chars requirment, create another one
-                    <text>if (!meetUniqueRule(password)) return generatePassword();</text>
+                // The generated password does not meet the required-unique-chars requirment, create another one
+                    <text>if (!meetUniqueRule(password)) return generatePassword(); </text>
```

Comment:
There's an extra space, also you could use ternary operator here

Reply:
all the code in `<text></text>` has that extra space. just so each line of code has a space after it.